meilisearch-release-changelog/v1.0.0.md

```diff
@@ -0,0 +1,122 @@
+📖 Meilisearch's official documentation will be ready for the official v1.0.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+# Charabia
+
+TBD
+
+# Generate master key option
+
+You can now take advantage of the `--generate-master-key` CLI option.
+
+By using that option, you can easily generate a strong master key to secure your Meilisearch instance easily. Using that option won't start Meilisearch and will exit without any error.
+
+Done in #3274 by @dureuill
+
+## Other improvements
+
+* Search speed improvements:
+  * Significantly improve the memory usage of search requests containing multiple long words (#3115) @loiclec
+  * Significantly improve the performance of the `exactness` ranking rule for search requests containing many words (#3116) @loiclec
+  * Translate multi-word synonyms into [phrases](https://docs.meilisearch.com/learn/what_is_meilisearch/features.html#phrase-search) when interpreting a search query. This change improves the relevancy of the results and stabilises the latency of search queries containing many multi-word synonyms, removing a source of denial-of-service attacks. (#3125) @loiclec
+  * Improve the performance of the `proximity` ranking rule for search requests ending with short words (#3118) @loiclec
+* You can now migrate from any old versions of Meilisearch (supporting the dump feature) to the latest version using a single dump. We hope to facilitate your migration to v1.0.0 so you can enjoy the latest features as easily as possible (#3170) @dureuill
+* Improve [`download-latest.sh`](https://github.com/meilisearch/meilisearch/blob/main/download-latest.sh) script: integrate apple-silicon binary (#3169) @curquiza
+* Clarify error message when the DB and engine versions are incompatible (#3279) @dureuill
+
+# Breaking changes ⚠️
+
+Since we are releasing v1.0.0, this is the last time we are introducing breaking changes to this release before v2.0.0.
+
+## New CLI behavior
+
+* For security reasons, Meilisearch will now reject master keys that are less than 16 bytes (#3274) @dureuill
+* Remove `--max-index-size` and `--max-task-db` configuration options (#3278) @dureuill
+Indeed, these options did not include the real space Meilisearch takes on the disk and were not adapted to the user's needs.
+* Remove `--disable-auto-batching` CLI option (#3296) @loiclec
+This option was initially introduced as a workaround in case of a bug and is no longer useful since the feature has matured.
+* Rename dump command from `--dumps-dir` to `--dump-dir` (#3175) @dureuill
+* Remove `--snapshot-interval-sec` and `--schedule-snapshot` now accepts an optional integer value specifying the interval in seconds (#3281) @dureuill
+* The `--log-level` option and `MEILI_LOG_LEVEL` environment variable now only accept the following arguments: `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE` and `OFF` (#3293) @loiclec
```

Comment:
```suggestion
* The `--log-level` option and `MEILI_LOG_LEVEL` environment variable accept the following arguments: `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE` and `OFF` (#3293) @loiclec
```

Reply:
It will change the phrase sentence, which is not what we want to say. Currently, Meilisearch can accept these values, but also any value, like strings, that Meilisearch can interpret to apply a filter to the logs.
We did not code anything, we only sent the string to `logger` aka the Rust library that handles the logs and which interprets the string.
We changed this, Meilisearch now only does what the documentation details, so Meilisearch now only accepts the following value (`ERROR`...)

Here is [the related issue](https://github.com/meilisearch/meilisearch/issues/3292) (no team are pinged because it does not impact anyone, except the users who made tricky and un-documented things)

Reply:
Got it. The original sentence makes more sense